WordPress/gutenberg
# Query Loop: Include a Query Pagination option on the block variations

## Description

Adds the query pagination block with its inner blocks to the query loop variations.

Closes https://github.com/WordPress/gutenberg/issues/35134

## How has this been tested?




1. Add a query loop.
2. Select the option "Start blank"
3. Test all 4 variations and confirm that the query pagination is added below the post template.

## Types of changes




Enhancement


## Checklist:
